Library/Formula/srmio.rb

```diff
@@ -15,7 +15,7 @@ class Srmio < Formula
 
   def install
     if build.head?
-      system "chmod u+x genautomake.sh"
+      chmod 1, "genautomake.sh"
```

Comment:
Hi guys - sorry, but I'm having trouble pushing my new commit. I've followed the steps in the [PR guide](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md#how-to-open-a-homebrew-pull-request-and-get-it-merged) but when I try to push, I keep getting the error below:
```fatal: unable to access 'https://github.com/Homebrew/homebrew/': The requested URL returned error: 403```

`git remote -v` returns:
```
eirinikos	https://github.com/eirinikos/homebrew.git (fetch)
eirinikos	https://github.com/eirinikos/homebrew.git (push)
origin	https://github.com/Homebrew/homebrew (fetch)
origin	https://github.com/Homebrew/homebrew (push)
```
Those URLs are correct, aren't they?

I've tried `git push --force origin update-srmio` and a few variations thereof.